crates/sui-indexer/src/apis/governance_api_v2.rs

```diff
@@ -229,31 +229,20 @@ async fn exchange_rates(
         let mut rates = vec![];
         for df in state
             .inner
-            .get_dynamic_fields_in_blocking_task(
+            .get_dynamic_fields_raw_in_blocking_task(
                 exchange_rates_id,
                 None,
                 exchange_rates_size as usize,
             )
             .await?
         {
-            let epoch: EpochId = bcs::from_bytes(&df.bcs_name).map_err(|e| {
-                sui_types::error::SuiError::ObjectDeserializationError {
-                    error: e.to_string(),
-                }
-            })?;
-
-            let exchange_rate: sui_types::sui_system_state::PoolTokenExchangeRate = state
-                .inner
-                .spawn_blocking(move |this| {
-                    sui_types::dynamic_field::get_dynamic_field_from_store(
-                        &this,
-                        exchange_rates_id,
-                        &epoch,
-                    )
-                })
-                .await?;
+            let dynamic_field = df
```

Comment:
So if I'm understanding correctly, we were making an extraneous fetch? It seems like the data is already present and just needs to be deserialized from the df fetched from get_dynamic_fields_raw_in_blocking_task

Reply:
Yeah when we were doing the initial fetch we already had all the data